docs/Release_Notes.md

```diff
@@ -1,10 +1,16 @@
-GFS V16.3.22 RELEASE NOTES
+GFS V16.3.23 RELEASE NOTES
 
 -------
 PRELUDE
 -------
 
-The Gulf of Mexico is renamed to the Gulf of America. This is a non-science change and does not impact model output.
+The GFS is updated for the following:
+
+* workflow and UFS_UTILS package updates to use the new AFWA global snow file due to the hemispheric snow files being phased out
+* updated GSI code and convinfo file for saildrone observations and to allow assimilation of GOES-19 AMVs
+* ww3_outp is improved for the wave point output
+* bufr station updates to three stations and the addition of 121 new stations
+* obsproc/v1.2.3
```

Comment:
Or should this be 1.2.4?

Reply:
This should be 1.2.4, thanks for catching my copy of the prior version into here! I'll correct this.

Reply:
Corrected @ 9367ac13c